lib/otel/constants.js

```diff
@@ -97,6 +97,13 @@ module.exports = {
    */
   ATTR_HTTP_ROUTE: 'http.route',
 
+  /**
+   * The url target/path of the request. Includes query.
+   *
+   * @example /foo
+   */
+  ATTR_HTTP_TARGET: 'http.target',
```

Comment:
so the spec compliant attrs are `url.query`, and `url.full`, if those don't exist, you can use `http.url` to parse everything you need. looks like undici is compliant and http _could_ be but an env var is required: these are the old attributes https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-instrumentation-http#server-spans-legacy